ocs_ci/ocs/perftests.py

```diff
@@ -1023,3 +1027,114 @@ def cleanup_testing_pod(self):
+        c_result, min_cpu, max_cpu, cpu_percentile_99, cpu_percentile_95 = (
+            {},
+            {},
+            {},
+            {},
+            {},
+        )
+        m_result, min_mem, max_mem, mem_percentile_99, mem_percentile_95 = (
+            {},
+            {},
+            {},
+            {},
+            {},
+        )
+        ocp_pod_obj = OCP(kind=constants.POD, namespace=cluster_namespace)
+        pods = pod1.get_all_pods(namespace=cluster_namespace)
+        end_time = int(time.time())
+        prometheus = PrometheusAPI()
+        CPU_USAGE_POD = (
+            "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate"
+        )
+        MEM_USAGE_POD = 'sum(container_memory_working_set_bytes{cluster="", container != "", image != "",'
+
+        for podd in pods:
+            pod_status = ocp_pod_obj.get_resource_status(podd.name)
+            if pod_status == constants.STATUS_RUNNING:
+                query1 = (
+                    "sum("
+                    + CPU_USAGE_POD
+                    + "{namespace= "
+                    + "'"
+                    + cluster_namespace
+                    + "'"
+                    + ",pod="
+                    + "'"
+                    + podd.name
+                    + "'"
+                    + "}) by (pod)"
+                )
+                cpu_result = prometheus.query_range(
+                    query=query1,
+                    start=start_time,
+                    end=end_time,
+                    step=15,
+                )
+                list_val_c = cpu_result[0]["values"]
+                cpu_val = [float(el[1]) for el in list_val_c]
+                c_result[podd.name] = cpu_val
+                min_cpu[podd.name] = min(cpu_val)
+                max_cpu[podd.name] = max(cpu_val)
+                cpu_percentile_99[podd.name] = np.percentile(cpu_val, 99)
+                cpu_percentile_95[podd.name] = np.percentile(cpu_val, 95)
+
+                mem_result = (
+                    prometheus.query_range(
+                        query=MEM_USAGE_POD
+                        + " namespace = "
+                        + "'"
+                        + cluster_namespace
+                        + "'"
+                        + ",pod = "
+                        + "'"
+                        + podd.name
+                        + "'"
+                        + "})",
+                        start=start_time,
+                        end=end_time,
+                        step=15,
+                    ),
+                )
+                list_val_m = mem_result[0][0]["values"]
+                mem_val = [float(el[1]) for el in list_val_m]
+                m_result[podd.name] = mem_val
+                min_mem[podd.name] = min(mem_val)
+                max_mem[podd.name] = max(mem_val)
+                mem_percentile_99[podd.name] = np.percentile(mem_val, 99)
+                mem_percentile_95[podd.name] = np.percentile(mem_val, 95)
+
```

Comment:
please, create one dictionary, with all of the collected data, and then dump it to a  JSON file, so in the next step we ca push this file to the ES, and use the data in the Performance dashboard.